### PR TITLE
cleanup tests and bash indentation

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

**Review by kanagarajkm — APPROVED** [review]

**Review by dvaldivia — CHANGES_REQUESTED** [review]
The output of serializing with `gopkg.in/yaml.v2` is radically different from the one from `sigs.k8s.io/yaml`, and the output is incompatible with `kubectl apply -f` so not good 

Screen Shot 2022-01-31 at 6 23 31 PM

**Review by dvaldivia — DISMISSED** [review]

**Comment by harshavardhana** [comment]
PTAL @dvaldivia @kanagarajkm

**Comment by kanagarajkm** [comment]
@harshavardhana prometheus is failing with following error

```
ts=2022-02-01T14:10:12.900Z caller=main.go:434 level=error msg="Error loading config (--config.file=/etc/prometheus/prometheus.yml)" err="parsing YAML file /etc/prometheus/prometheus.yml: yaml: unmarshal errors:\n  line 4: field Global not found in type config.plain\n  line 7: field ScrapeConfigs not found in type config.plain"
```

```
apiVersion: v1
data:
  prometheus.yml: |
    # This file and config-map is generated by MinIO Operator.
    # DO NOT EDIT.

    Global:
      EvaluationInterval: 30000000000
      ScrapeInterval: 30000000000
    ScrapeConfigs:
    - BearerToken: eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJleHAiOjQ3OTczMjQ0MjIsImlzcyI6InByb21ldGhldXMiLCJzdWIiOiIwSUkwS0lMVjJLWldZVFFKIn0.GUY1Syy4S02imTlfldA5Dxnrw45joUCALYXF3s_rFk4SVYC293oPrcCtbH1J-N2KxS9HyvWX0VWGssQ2VcUcpg
      JobName: minio-job
      MetricsPath: /minio/v2/metrics/cluster
      Scheme: https
      StaticConfigs:
      - Targets:
        - minio.tenant1.svc.cluster.local:443
      TLSConfig:
        CAFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
kind: ConfigMap
metadata:
  creationTimestamp: "2022-02-01T14:07:02Z"
  name: tenant1-prometheus-config-map
  namespace: tenant1
  ownerReferences:
  - apiVersion: minio.min.io/v2
    blockOwnerDeletion: true
    controller: true
    kind: Tenant
    name: tenant1
    uid: 0a5c90aa-44a7-4ec9-80b1-e2236ae0b35d
  resourceVersion: "160755"
  uid: e2aea3cb-7119-42e0-b4eb-5314220bb8d1
```

**Comment by harshavardhana** [comment]
> @harshavardhana prometheus is failing with following error
> 
> ```
> ts=2022-02-01T14:10:12.900Z caller=main.go:434 level=error msg="Error loading config (--config.file=/etc/prometheus/prometheus.yml)" err="parsing YAML file /etc/prometheus/prometheus.yml: yaml: unmarshal errors:\n  line 4: field Global not found in type config.plain\n  line 7: field ScrapeConfigs not found in type config.plain"
> ```
> 
> ```
> apiVersion: v1
> data:
>   prometheus.yml: |
>     # This file and config-map is generated by MinIO Operator.
>     # DO NOT EDIT.
> 
>     Global:
>       EvaluationInterval: 30000000000
>       ScrapeInterval: 30000000000
>     ScrapeConfigs:
>     - BearerToken: eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJleHAiOjQ3OTczMjQ0MjIsImlzcyI6InByb21ldGhldXMiLCJzdWIiOiIwSUkwS0lMVjJLWldZVFFKIn0.GUY1Syy4S02imTlfldA5Dxnrw45joUCALYXF3s_rFk4SVYC293oPrcCtbH1J-N2KxS9HyvWX0VWGssQ2VcUcpg
>       JobName: minio-job
>       MetricsPath: /minio/v2/metrics/cluster
>       Scheme: https
>       StaticConfigs:
>       - Targets:
>         - minio.tenant1.svc.cluster.local:443
>       TLSConfig:
>         CAFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
> kind: ConfigMap
> metadata:
>   creationTimestamp: "2022-02-01T14:07:02Z"
>   name: tenant1-prometheus-config-map
>   namespace: tenant1
>   ownerReferences:
>   - apiVersion: minio.min.io/v2
>     blockOwnerDeletion: true
>     controller: true
>     kind: Tenant
>     name: tenant1
>     uid: 0a5c90aa-44a7-4ec9-80b1-e2236ae0b35d
>   resourceVersion: "160755"
>   uid: e2aea3cb-7119-42e0-b4eb-5314220bb8d1
> ```

Yes thanks @kanagarajkm this is what I was chasing.

File: kubectl-minio/cmd/proxy.go [hunk 1]
```diff
@@ -158,8 +157,8 @@ func servicePortForwardPort(ctx context.Context, namespace, service, port string
 			log.Printf("failed to capture stdout or stderr\n")
 			return
 		}
-		//outStr, errStr := string(stdout), string(stderr)
-		//fmt.Printf("\nout:\n%s\nerr:\n%s\n", outStr, errStr)
+		// outStr, errStr := string(stdout), string(stderr)
+		// fmt.Printf("\nout:\n%s\nerr:\n%s\n", outStr, errStr)
```
Comment (Alevsk) [inline kubectl-minio/cmd/proxy.go:161]: we can delete commented code

File: pkg/controller/cluster/http_handlers.go [hunk 2]
```diff
@@ -236,7 +234,7 @@ func (c *Controller) CRDConversionHandler(w http.ResponseWriter, r *http.Request
 		case "v1":
 
 			// cast to v1
-			//tenantV1 := obj.Object.(*miniov1.Tenant)
+			// tenantV1 := obj.Object.(*miniov1.Tenant)
```
Comment (Alevsk) [inline pkg/controller/cluster/http_handlers.go:237]: delete commented code

File: pkg/controller/cluster/main-controller.go [hunk 3]
```diff
@@ -792,7 +790,7 @@ func (c *Controller) syncHandler(key string) error {
 	}
 
 	// Check if this is fresh setup not an expansion.
-	//addingNewPool := len(tenant.Spec.Pools) == len(tenant.Status.Pools)
+	// addingNewPool := len(tenant.Spec.Pools) == len(tenant.Status.Pools)
```
Comment (Alevsk) [inline pkg/controller/cluster/main-controller.go:793]: delete commented code

**Review by Alevsk — APPROVED** [review]

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**